CONTRIBUTING.md

```diff
@@ -66,10 +66,14 @@ The following section explains various suggestions and procedures to note during
 
 ### Prerequisites
 
-* It is strongly recommended that you use Linux distributions systems or OSX for development.
+* It is strongly recommended that you use Linux distributions systems or macOS for development.
+* Running WSL2 on Windows is also possible. Note that if during development you run a local Kubernetes cluster and have a Service with `service.spec.sessionAffinity: ClientIP`, it will break things until it's removed[^windows_xt_recent]. 
 * Go 1.19.x or higher.
+* Docker (to run e2e tests)
 * For React UI, you will need a working NodeJS environment and the npm package manager to compile the Web UI assets.
 
+[^windows_xt_recent]: A WSL 2 kernel recompilation is required to enable the `xt_recent` kernel module, used by `iptables` in `kube-proxy` to implement ClientIP session affinity. See [issue in WSL](https://github.com/microsoft/WSL/issues/7124).
```

Comment:
FYI this is a markdown footnote and Github Markdown supports/renders it nicely.

Reply:
Yup, I see it renders in deploy preview too: https://deploy-preview-5927--thanos-io.netlify.app/tip/contributing/contributing.md/#fn:1